src/sympc/protocol/falcon/falcon.py

```diff
@@ -71,6 +73,82 @@ def __eq__(self, other: Any) -> bool:
 
         return True
 
+    @staticmethod
+    def trunc1(
```

Comment:
Q: If this operation is specific to ABY3 could we move it to a folder named `aby3` in `protocols` folder (as we have with falcon). After that, we can call that function here by `ABY3.truncate`.

Reply:
Sure George, I would change it.